include/envoy/server/instance.h

```diff
@@ -116,6 +117,11 @@ class Instance {
    */
   virtual ListenerManager& listenerManager() PURE;
 
+  /**
+   * @return the server's global mutex tracer.
+   */
+  virtual absl::optional<Envoy::MutexTracer*> mutexTracer() PURE;
```

Comment:
nit: Sorry just noticed this. I think you can do away with the optional here and just return the pointer since nullptr would indicate no tracer? Thoughts?

Reply:
This makes sense to me. Implemented in 022fd8e.